### PR TITLE
Miscs

### PR DESCRIPTION
**Pull request by nsurbay**
- Change version number to support element greater than 9
- Update docker and package images
- Use doxygen 1.9.2 in readthedocs

### CODE REVIEW

File: docker/ubuntu_debian/Dockerfile [hunk 1]
```diff
@@ -32,12 +31,7 @@ RUN apt-get update && \
     rm -rf /var/lib/apt/lists/*
 
 # create a user
-RUN adduser --disabled-password --gecos '' $USER && \
-    adduser $USER sudo && \
-    echo '%sudo ALL=(ALL) NOPASSWD:ALL' >> /etc/sudoers
-
-# switch to new user
-USER $USER
+RUN adduser --disabled-password --gecos '' $USER
```
Comment (toffan) [inline docker/ubuntu_debian/Dockerfile:34]:
```suggestion
RUN adduser --disabled-password --gecos '' --home $HOME $USER
```

File: docker/ubuntu_debian/Dockerfile.ubuntu18_04 [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+        python3 \
+        python3-pip \
+        python3-dev && \
+    python3 -m pip install cmake && \
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/*
+
+# create a user
+RUN adduser --disabled-password --gecos '' $USER --home $HOME && \
+    chown -R $USER:$USER $HOME
```
Comment (toffan) [inline docker/ubuntu_debian/Dockerfile.ubuntu18_04:36]: I don't understand why is there a `chown` here and not in the other Dockerfiles.

File: .readthedocs.yml [hunk 3]
```diff
@@ -14,7 +18,5 @@ formats:
 python:
   version: 3.8
   install:
-    - requirements: docs/requirements.txt
-    - requirements: docs/requirements.readthedocs.txt
```
[the remark this answers is not in the document]
Comment (nsurbay) [reply .readthedocs.yml:18]: The requirement file are needed to setup your local env. However, I can merge them.

File: docker/ubuntu_debian/Dockerfile.ubuntu18_04 (hunk 2 above)
Comment (nsurbay) [reply docker/ubuntu_debian/Dockerfile.ubuntu18_04:36]: A better solution can be to create the user before running apt.

**Pull request merged**